_lessons/python-scatter/02-vectorising.md

```diff
@@ -37,18 +37,22 @@ Good candidates are loops where the same function is applied to each element. Re
 
 When considering nested loops, start by vectorising the innermost loop, unless the innermost loop only performs very few iterations.
 
-Array operations are available through the `numpy` Python module. `numpy` arrays in many respects behave like lists with the following caveats:
+## Use numpy for fast array operations
+
+ Array operations are available through the `numpy` Python module. `numpy` arrays in many respects behave like lists with the following differences:
 
  * all array elements must have the **same type** (integer, float, etc.)
  * array elements cannot be added or removed (without having to recreate the array)
 
-On the other hand, `numpy` arrays support elementwise operations.
+On the other hand, `numpy` arrays support elementwise and reduction operations.
+
+Vectorised Python code using large `numpy` arrays should almost always run much faster than plain Python code with loops, and it can run as fast as C code in some cases. 
```

Comment:
Compiled instead of C? Maybe not worth it

Reply:
Thanks @chrisdjscott.